config/initializers/qualifications/a_and_as_level.rb

```diff
@@ -0,0 +1,187 @@
+A_AND_AS_LEVEL_SUBJECTS = [
```

Comment:
@paulrobertlloyd would appreciate a review of this list in particular to see if:

- there are any edits required for formatting consistency
- whether anything can be deleted/merged. There's some potential repetition in, eg - Biology, Chemistry, Physics, and Design and Technology.

Reply:
👍  seems worth de-duplicating these.

Reply:
I think we should stick with this subject list for now, with all its subject/exam variations.